tests/queries/tests.py

```diff
@@ -1216,6 +1216,9 @@ def test_field_with_filterable(self):
             [self.a3, self.a4],
         )
 
+    def test_filter_by_negate_field(self):
+        Note.objects.filter(negate=True)
+
```

Comment:
It wouldn't hurt to evaluate the queryset to make sure it doesn't break somewhere down the line.